drivers/benchmark_graphics.cpp

```diff
@@ -27,7 +28,11 @@ int main(int argc, char *argv[]) {
     }
     auto scene = build_scene(context, std::move(objects));
 
-    while (!should_exit(context)) {
-        render_frame(context, scene);
+    while (!window->should_close()) {
+        window->poll_events();
+        camera->handle_keys(0.0001f);
```

Comment:
Where is camera supposed to be defined?

Reply:
Camera should be created separately, after creating the context. I just pushed an example to ``ivs.cpp``.